docs/checks/internal/validate/main.go

```diff
@@ -140,6 +140,10 @@ func supportedInterfacesFromImplementation(checkName string, checkFiles map[stri
 		return []string{"GitHub"}, nil
 	}
 
+	if checkName == "LicenseCheck" {
```

Comment:
Previously running

`go run ./docs/checks/internal/validate/main.go`

was getting the following error

```
panic: validateRepoTypeAPIs: LicenseCheck: got diff:   []string{
- 	"local",
  	"GitHub",
  }
```

After adding this changes in it fix the problem

@laurentsimon Not sure if I'm fixing this the right way, but after adding this not getting error for the `checks.yaml`.

Reply:
please remove the `if statement` you added. You caught a validation check and I'm glad you did as it shows it works (we added it last week)!
The right way to fix it to edit the checks.yaml file, see example in https://github.com/ossf/scorecard/blob/main/docs/checks/internal/checks.yaml#L47

The `repos` flags is a list of repo types that the check supports. In your case, it works on local files and for GitHub repos, so just add `GitHub, local`. The run `make generate-docs`. If should validate the files, re-generate the checks.md. Add both to your PR

Reply:
@laurentsimon Updated the PR. Thanks.